Formula/tomee-webprofile.rb

```diff
@@ -15,7 +18,11 @@ def install
     # Install files
     prefix.install %w[NOTICE LICENSE RELEASE-NOTES RUNNING.txt]
     libexec.install Dir["*"]
-    bin.install_symlink "#{libexec}/bin/startup.sh" => "tomee-webprofile-startup"
+    libexec.install_symlink "#{libexec}/bin/startup.sh" => "tomee-webprofile-startup"
```

Comment:
Is the intermediate symlink needed? seems like env script could exec startup.sh itself

Reply:
Not needed; this way just made more sense to me design-wise, separating the name indirection from the environment wrapping. I can condense it if you prefer.

Reply:
seems fine as you have it I suppose